src/authorizations/actions/thunks.ts

```diff
@@ -166,3 +167,18 @@ export const deleteAuthorization = (id: string, name: string = '') => async (
     dispatch(notify(authorizationDeleteFailed(name)))
   }
 }
+
+export const getResources = () => async dispatch => {
```

Comment:
I'm a little confused about the intent of this - what does this do that the current `GetResources` component doesn't?

Reply:
@hoorayimhelping `GetResources` component takes an array of `ResourceTypes` as a prop and only fetches those resources. This new `getResources` function is fetching all resource names from the new api route that was created. Hope that makes sense!

Reply:
@Palakp41 that makes sense. What do you think about naming this `getAllResources` so it's clear we're fetching them all?